templates/taxi/car_list.html

```diff
@@ -1,16 +1,38 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a style="float: right" href="{% url 'taxi:car-create' %}">+</a></h1>
```

Comment:
It would be better to have a separate button for creating a new car. Styling inline elements with `float: right` is not a good practice. It's better to use CSS classes for styling.